en/hardware/porting_guide_config.md

```diff
@@ -30,6 +30,11 @@ if DRIVERS_UAVCAN_V1
 endif #DRIVERS_UAVCAN_V1
 ```
 
+:::note
+The `menuconfig` property in Kconfig file is the name used for configuration.
```

Comment:
This is a bit confusing. Perhaps we should move this line as a note next to the above comment "This configuration is handled through *Kconfig*, ...." ?